app/cdap/components/ConnectionList/index.tsx

```diff
@@ -201,62 +201,69 @@ export default function ConnectionList() {
+              }
+              if (index === 0) {
+                headerContent = headerForLevelZero();
+              } else {
+                headerContent =
+                  refs.current[index]?.offsetWidth < refs.current[index]?.scrollWidth ? (
+                    <CustomTooltip title={dataForTabs[index - 1].selectedTab} arrow>
+                      <Box className={classes.beforeSearchIconClickDisplay}>
+                        <Typography
+                          variant="body2"
+                          className={classes.headerLabel}
+                          ref={(element) => {
+                            refs.current[index] = element;
+                          }}
+                        >
+                          {dataForTabs[index - 1].selectedTab}
+                        </Typography>
+                      </Box>
+                    </CustomTooltip>
+                  ) : (
+                    <Box className={classes.beforeSearchIconClickDisplay}>
+                      <Typography
+                        variant="body2"
+                        className={classes.headerLabel}
+                        ref={(element) => {
+                          refs.current[index] = element;
+                        }}
+                      >
+                        {dataForTabs[index - 1].selectedTab}
+                      </Typography>
+                    </Box>
+                  );
+              }
+              return (
+                <Box className={classes.tabsContainerWithHeader}>
+                  <Box className={classes.tabHeaders}>{headerContent}</Box>
+                  <ConnectionsTabs
+                    tabsData={each}
+                    handleChange={selectedTabValueHandler}
+                    value={each.selectedTab}
+                    index={index}
+                    connectionId={connectionId || ''}
+                    toggleLoader={(value: boolean, isError?: boolean) =>
+                      toggleLoader(value, isError)
+                    }
+                    setIsErrorOnNoWorkSpace={setIsErrorOnNoWorkSpace}
+                  />
                 </Box>
               );
-          }
-          return (
-            <Box className={classes.tabsContainerWithHeader}>
-              <Box className={classes.tabHeaders}>{headerContent}</Box>
-              <ConnectionsTabs
-                tabsData={each}
-                handleChange={selectedTabValueHandler}
-                value={each.selectedTab}
-                index={index}
-                connectionId={connectionId || ''}
-                toggleLoader={(value: boolean, isError?: boolean) => toggleLoader(value, isError)}
-                setIsErrorOnNoWorkSpace={setIsErrorOnNoWorkSpace}
-              />
-            </Box>
-          );
-        })}
-      </SelectDatasetWrapper>
-
+            })}
+          </SelectDatasetWrapper>
+        </>
+      ) : (
+        <h6 className={classes.noDataLabel}>No Connections to show</h6>
```

Comment:
use translation here

Reply:
It's a temporary placeholder for now, It will gonna be removed in the upcoming branches